Readme.md

```diff
@@ -0,0 +1,172 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>Budget app</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget app ](#-budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Install this project with:](#install-this-project-with)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget app <a name="about-project"></a>
+
+The Ruby on Rails capstone project involves developing a mobile web application for budget management. In this project, you will build a Ruby on Rails application that enables users to:
+
+- Register and log in to maintain the privacy of their data.
+- Add new financial transactions linked to specific categories.
+- Track and visualize their expenses for each category, providing insights into their spending habits.
+
+## 🛠 Built With <a name="built-with"></a>
+Ruby on Rails
+
+### Tech Stack <a name="tech-stack"></a>
+
+- <a href="https://www.postgresql.org/">PostgreSQL</a>
+- <a href="https://rubyonrails.org/">Ruby on Rails</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+Key features of the application 🦖
+
+- User Registration & Authentication
+- Category & Budget Management
+- Transaction Management
+- Category & Transaction Creation
+- Testing (Unit & Integration)
+- Technical Stack (PostgreSQL, Devise)
+- Security Validation
+- Online Deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Coming Soon [Live Demo Link]()
```

Comment:
Nice job so far👍🏼. Kindly do well to add the link to the live version of your project as it is a requirement for this milestone that our project is deployed and the link to the deployed project is added to the readme file😉.

<img width="793" alt="image" src="https://github.com/Rifat8080/budget-app/assets/64643995/77a13004-1375-41fb-b320-cbb47fd002af">